servers/credentialagent/src/controllers/operator/tenants/_tenantId/controller-v0.8.js

```diff
@@ -153,9 +153,6 @@ const tenantController = async (fastify) => {
       await repos.vendorUserIdMappings
         .collection()
         .deleteMany({ tenantId: tenant._id });
-      await repos.revocationListAllocations.collection().deleteMany({
```

Comment:
Why are we not replacing this with `allocations` collection removal? They are by tenant, so if we make another tenant with same keys, they will have same operator address. I think they are still guarded with a tenant Id so it should be fine.

Also is it a mistake that metadata allocations wasn't being deleted?

Should tests be updated?

Reply:
fixed